integration/e2e/e2e_test.go

```diff
@@ -179,15 +181,12 @@ var _ = Describe("EndToEnd", func() {
 			nwo.DeployChaincode(network, "testchannel", orderer, chaincode)
 
 			By("ensuring external cc run artifacts exist after deploying")
-			runArtifactsFile, err := os.Open(runArtifactsFilePath)
+			contents, err := ioutil.ReadFile(runArtifactsFilePath)
```

Comment:
One thing that should probably be added is a check to ensure that contents is not empty.